index.html

```diff
@@ -1477,57 +1503,59 @@
 
 function removeContext(patch) {
     const hunks = new Array();
-    let srcStart = 0;
-    let srcLines = new Array();
-    let dstStart = 0;
-    let dstLines = new Array();
-    let isBefore = true;
-
-    const lines = patch.split('\n');
-    const end = patch.endsWith('\n') ? lines.length - 1 : lines.length;
-    for (let i = 0; i < end; i++) {
-        const line = lines[i];
-        if (line.startsWith("@@")) {
-            if (srcLines.length > 0 || dstLines.length > 0) {
+    if (patch != null) {
```

Comment:
Maybe do an early return instead? As in

```javascript
if (patch === null) {
    return hunks;
}
```

That way you don't have to re-indent the entire method 😄